src/command_modules/azure-cli-security/azure/cli/command_modules/security/_client_factory.py

```diff
@@ -0,0 +1,65 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def _cf_security(cli_ctx, **_):
+    from azure.cli.core.commands.client_factory import get_mgmt_service_client
+    from azure.mgmt.security import SecurityCenter
+
+    clt = get_mgmt_service_client(cli_ctx, SecurityCenter, asc_location="centralus")
+
+    return clt
```

Comment:
Combine the two lines please. Also, why are we specifying centralus for the location.

Reply:
we have a task to remove the asc_location from the initialization of the clients. currently we hide the asc_location usage from the user.
centralus is a just arbitrary value and is our most common region.